src/window/virtual-line.lisp

```diff
@@ -1,14 +1,23 @@
 (in-package :lem-core)
 
-(defun window-recenter (window)
-  (unless (= (window-cursor-y window)
-             (floor (window-height-without-modeline window) 2))
+(defun window-recenter (window &key (line nil) (from-bottom nil))
```

Comment:
Hi, my 2c: &key args are NIL by default already, and I don't think Lem's codebase writes nils explicitely like this. 

thanks for the feature.

Reply:
Sure, I removed the default value of the keyword arguments.